terraform/paas/resources.tf

```diff
@@ -0,0 +1,31 @@
+resource cloudfoundry_app publish-training {
+  name         = var.app.name
+  space        = data.cloudfoundry_space.space.id
+  docker_image = var.app.docker_image
+  strategy     = "blue-green-v2"
+
+  environment = {
+    ASSETS_PRECOMPILE                                = var.app_env.ASSETS_PRECOMPILE
+    RAILS_ENV                                        = var.app_env.RAILS_ENV
+    RAILS_SERVE_STATIC_FILES                         = var.app_env.RAILS_SERVE_STATIC_FILES
+    SECRET_KEY_BASE                                  = var.SECRET_KEY_BASE
+    SENTRY_DSN                                       = var.SENTRY_DSN
+    SETTINGS__GOOGLE__GCP_API_KEY                    = var.SETTINGS__GOOGLE__GCP_API_KEY
```

Comment:
The `SETTINGS__GOOGLE__GCP_API_KEY` key isn't used by Publish.

Reply:
removed